IndyAV/control/indyav_control/msg/VelocityStamped.msg

```diff
@@ -0,0 +1,2 @@
+Header header
+float64 velocity
```

Comment:
definately either rename this feild to `radians_per_second` or `rotational_velocity` and leave a comment about how this is in `radians per second`